dynamic_rest/fields/generic.py

```diff
@@ -84,7 +84,7 @@ def to_representation(self, instance):
             # We want the pk to be represented as an object with type,
             # rather than just the ID.
             pk_value = self.get_pk_object(
-                serializer_class().get_plural_name(),
+                serializer_class().get_name(),
```

Comment:
Let's change this to `serializer_class.get_name()` since it's a [class method](https://github.com/AltSchool/dynamic-rest/blob/master/dynamic_rest/serializers.py#L222) and it'll save us the overhead of constructing a serializer.